dyn_em/module_initialize_real.F

```diff
@@ -4333,11 +4333,7 @@ SUBROUTINE init_domain_rk ( grid &
          do i = its, min(ide-1,ite)
             z1 = (grid%phb(i,2,j)-grid%phb(i,1,j))/g
             airmass = 1./grid%alt(i,1,j) * z1 * config_flags%dx*config_flags%dy          ! kg
```

Comment:
airmass no longer needed

Reply:
Jimy,
this code has been changed based on your suggestion.